images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -103,6 +103,7 @@ $tools.AddToolVersion("Bazelisk", $(Get-BazeliskVersion))
 $tools.AddToolVersion("Bicep", $(Get-BicepVersion))
 $tools.AddToolVersion("Buildah", $(Get-BuildahVersion))
 $tools.AddToolVersion("CMake", $(Get-CMakeVersion))
+$tools.AddToolVersion("CodeQL Action Bundle", $(Get-PriorCodeQLBundleVersion))
 $tools.AddToolVersion("CodeQL Action Bundle", $(Get-CodeQLBundleVersion))
```

Comment:
I don't see other examples where we call `AddToolVersion` with the same tool name but different versions. Just want to be sure this will append the second version rather than overwriting the first version.

Reply:
Oh, nice catch, I think you're right. Looks like `AddToolVersion` is defined [here](https://github.com/actions/runner-images/blob/9ee93111224473826a8168cc6b2951f11fdba197/helpers/software-report-base/SoftwareReport.Nodes.psm1#L79) which calls [`AddNode`](https://github.com/actions/runner-images/blob/9ee93111224473826a8168cc6b2951f11fdba197/helpers/software-report-base/SoftwareReport.Nodes.psm1#L41), which in turns throws if the node name has been found. 

I'll update the name.